keyboards/handwired/mingwahlu/mingwahlumacro/rules.mk

```diff
@@ -0,0 +1,23 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+BOOTLOADER = atmel-dfu
```

Comment:
```suggestion
```

Reply:
kept BOOTLOADER = atmel-dfu and removed BOOTLOADER = caterina